shepherd_bms/src/analyzer.cpp

```diff
@@ -384,4 +385,18 @@ void Analyzer::disableTherms()
             }
         }
     }
+}
+
+void Analyzer::calcStateOfCharge()
+{
+    int index = (((bmsdata->minVoltage.val) - MIN_VOLT) / .1);
+
+    // .1 = 1.3V range / 13 datapoints on curve
```

Comment:
This should be mentioned in the comments with the LUT. Important to know for setting the LUT/curve